sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -4199,6 +4199,18 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val LEGACY_AVRO_ALLOW_READING_WITH_INCOMPATIBLE_SCHEMA =
+    buildConf("spark.sql.legacy.avro.allowReadingWithIncompatibleSchema")
```

Comment:
Do we plan to forbid the writing as well? If yes, shall we use one config `spark.sql.legacy.avro.allowIncompatibleSchema` for both read and write?

Reply:
Sure, it make sense to me.